deployment/ansible/group_vars/all

```diff
@@ -1,3 +1,5 @@
 ---
+develop: "'development' in group_names"
```

Comment:
Good to know. Custom filters seem like overkill for this size of project, so I'll probably fall back to inserting these via group_vars.